frontend/public/locales/en/apply-adult-child.json

```
@@ -634,11 +634,9 @@
     "continue-btn": "Continue"
   },
   "parent-or-guardian": {
-    "page-title": "Parent or legal guardian needs to apply",
-    "unable-to-apply-child": "Your child is not currently eligible to apply for the Canadian Dental Care Plan at this time.",
-    "unable-to-apply": "You are not able to apply for the Canadian Dental Care Plan for yourself. Please have a parent or legal guardian apply on your behalf.",
-    "service-canada": "If you have already applied for your child and now want to apply for yourself, you must speak to a Service Canada representative. Contact Service Canada by calling <noWrap>1-833-537-4342</noWrap>.",
-    "eligibility": "The Canadian Dental Care Plan is currently accepting applications for people aged 65 and above, children under 18, and adults with a valid disability tax credit.",
+    "page-title": "Parent or guardian needs to apply",
```

Should be left as is according to Figma:

```suggestion
    "page-title": "Parent or legal guardian needs to apply",
```